cinder/volume/drivers/nvme.py

```diff
@@ -109,19 +108,22 @@ def get_volume_stats(self, refresh=False):
             'QoS_support': False}
 
         data['pools'].append(single_pool)
+
         return data
 
-    def create_export(self, context, volume, connector):
-        return {
-            'target_portal': self.configuration.target_ip,
-            'nqn':  volume['provider_location'],
-            'target_port': self.configuration.target_port
-        }
+    def initialize_connection(self, volume, connector):
+        return self.target_driver.initialize_connection(volume, connector)
+
+    def create_export(self, context, volume, connector, vg=None):
+        if vg is None:
```

Comment:
It's copy-paste from LVM driver and it's totally wrong for NVMe stuff. We don't need 'vg' param here.